fastlane/lanes/build.rb

```diff
@@ -135,30 +135,39 @@
   end
 
   #####################################################################################
-  # download_signed_apk_from_google_play
+  # download_signed_apks_from_google_play
   # -----------------------------------------------------------------------------------
-  # This lane downloads the signed apk from Play Store for the given version and app
+  # This lane downloads the signed apks from Play Store for the given app and version
+  #
+  # If no argument is provided, it'll download both WordPress & Jetpack apks using the version from version.properties
+  # If only 'app' argument is provided, it'll download the apk for the given app using the version from version.properties
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane download_signed_apk_from_google_play app:<wordpress|jetpack> version:<versionName,versionCode>
+  # bundle exec fastlane download_signed_apks_from_google_play # Download WordPress & Jetpack apks using the version from version.properties
+  # bundle exec fastlane download_signed_apks_from_google_play app:<wordpress|jetpack> # Download given app's apk using the version from version.properties
+  # bundle exec fastlane download_signed_apks_from_google_play app:<wordpress|jetpack> version:<versionName,versionCode>
   #####################################################################################
-  lane :download_signed_apk_from_google_play do |options|
-    app = get_app_name_option!(options)
-    package_name = APP_SPECIFIC_VALUES[app.to_sym][:package_name]
-    version = options[:version] || android_get_release_version() # default to current release version
+  lane :download_signed_apks_from_google_play do |options|
+    # If no `app:` is specified, call this for both WordPress and Jetpack
+    apps = options[:app].nil? ? %i[wordpress jetpack] : Array(options[:app]&.downcase&.to_sym)
+
+    apps.each do |app|
+      package_name = APP_SPECIFIC_VALUES[app.to_sym][:package_name]
+      version = options[:version] || android_get_release_version() # default to current release version
+
+      if version.is_a?(String) # for when calling from command line
+        (version_name, version_code) = version.split(',')
+        UI.user_error!('Please pass the `version` option as a comma-separated `name,code` value') if version_code.nil?
+        version = { 'name' => version_name, 'code' => version_code }
+      end
```

Comment:
I just realized that this could technically be moved out of the loop since it doesn't depend on app so no need to re-compute it on each iteration 🙃

Reply:
🤦 

I already merged the PR, so I made the change directly in `release/23.0`: 5ba017012345536cdd4f5cd80c1d472f77ee1b24